doc-content/enterprise-only/openshift/template-deploy-amq-proc.adoc

```diff
@@ -0,0 +1,27 @@
+[id='template-deploy-amq-{context}-proc']
+= Configuring communication with an AMQ server for {what_deploying_brief}
+
+If you use the `{PRODUCT_INIT}{ENTERPRISE_VERSION_SHORT}-prod-immutable-kieserver-amq.yaml` template file, the {KIE_SERVER} is configured to receive and respond to API messages using JMS, communicating with an AMQ server.
```

Comment:
Change to something like:
"..., the JMS capabilities will be configured allowing you to interact with KIE_SERVER through JMS API."

remove communicating with...